api/common/interfaces.go

```diff
@@ -389,10 +389,10 @@ func getVersion(s Serverer, params map[string]interface{}) map[string]interface{
 	return respPacking(SUCCESS, config.Version)
 }
 
-// commitPor send por transaction
+// sendSigChain send por transaction
 // params: {"sigchain":<sigchain>}
 // return: {"resultOrData":<result>|<error data>, "error":<errcode>}
-func commitPor(s Serverer, params map[string]interface{}) map[string]interface{} {
+func sendSigChain(s Serverer, params map[string]interface{}) map[string]interface{} {
```

Comment:
Is this APi necessary in future? It just for test.

Reply:
I don't think so, it's never been used expect for testing